MoltenVK/MoltenVK/GPUObjects/MVKImage.mm

```diff
@@ -412,8 +412,12 @@
     if (_deviceMemory) { _deviceMemory->removeImageMemoryBinding(this); }
     MVKResource::bindDeviceMemory(mvkMem, memOffset);
 
+    if (!_deviceMemory) {
+        return VK_SUCCESS;
+    }
```

Comment:
MoltenVK style typically has this kind of trivial `if` statement on a single line. See for instance line 412 immediately above. Can you compress it please?